graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLResponseBody.java

```diff
@@ -51,7 +51,7 @@ public GraphQLResponseBody(Map<String, Object> specification) {
      * @return the GraphQL response body data
      */
     @JsonAnyGetter
```

Comment:
Should we remove `@JsonAnyGetter` if we are going to annotate with `@JsonIgnore`?

Reply:
No, I don't think so. Those two annotations serve different purposes.